crates/cli/src/subcommands/identity.rs

```diff
@@ -585,6 +577,7 @@ async fn exec_token(config: Config, args: &ArgMatches) -> Result<(), anyhow::Err
 async fn exec_set_name(mut config: Config, args: &ArgMatches) -> Result<(), anyhow::Error> {
     let new_name = args.get_one::<String>("name").unwrap();
     let identity_or_name = args.get_one::<String>("identity").unwrap();
+    config.can_set_name(Some(identity_or_name.as_str()))?;
```

Comment:
This is a regression. It essentially renders unnamed identities unusable.
```
   $ spacetime identity new -s local --no-email | tee output
    IDENTITY     631d3896d10a5fa9db73a95a5a2e20981ab71bc32cb908859e449b5eeb886e9a
    NAME         
    EMAIL        
 
   $ cat output | grep 'IDENTITY' | awk '{print $2;}' | tee identity
   631d3896d10a5fa9db73a95a5a2e20981ab71bc32cb908859e449b5eeb886e9a
 
   $ spacetime identity set-name $(cat identity) new-identity-name
-  Updated identity:
-   IDENTITY  631d3896d10a5fa9db73a95a5a2e20981ab71bc32cb908859e449b5eeb886e9a
-   NAME      new-identity-name
+  Error: An identity name cannot be an identity.
+  [1]
```

Reply:
Thank you for catching this Zeke that would have been a disaster 😦